provisioner/terraform/testdata/multiple-agents/multiple-agents.tfplan.json

```diff
@@ -1,6 +1,6 @@
 {
   "format_version": "1.1",
-  "terraform_version": "1.3.3",
+  "terraform_version": "1.3.6",
```

Comment:
We shouldn't change these manually, they are generated via `provisioner/terraform/testdata/generate.sh`. It's probably fine to leave them as-is without a bump.